src/test/java/com/sfuentes/StudentTest.java

```diff
@@ -0,0 +1,92 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StudentTest {
+
+  @Test
+  void sortByWomenAndMen_TenStudents_OnlyWomen() {
+    String[][] students = {
+        {"Daniela", "woman"},
+        {"Julia", "woman"},
+        {"Maria", "woman"},
+        {"Manuela", "woman"},
+        {"Luisa", "woman"},
+        {"Ana", "woman"},
+        {"Sonia", "woman"},
+        {"Luisa", "woman"},
+        {"Mariana", "woman"},
+        {"Sara", "woman"}
+    };
+    Student test = new Student(students);
+    Map<String, Integer> expected = new HashMap<>();
+    expected.put("Women: ", 10);
+    expected.put("Men: ", 0);
+    Map<String, Integer> actualResult = test.getCountByGender(students);
+
+    assertEquals(expected, actualResult);
```

Comment:
Here instead using a variable to store the actualResult you should use the method call directly.
Remember, always try to make your assertion more readable,
And do the same to the other assertions in this file.

Reply:
fixed